.gitignore

```diff
@@ -43,7 +43,7 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
+/node_modules
```

Comment:
Why was this changed from `node_modules/` to `/node_modules`?

Reply:
When i pulled develop i saw node modules folder active on my project. Not git ignored. When i made the change then the whole node modules folder was git ignored